src/log.c

```diff
@@ -174,7 +174,8 @@ void DEFAULT_SYMVER_PRE(fi_log)(const struct fi_provider *prov, enum fi_log_leve
 
 	va_list vargs;
 
-	size = snprintf(buf, sizeof(buf), "%s:%s:%s:%s():%d<%s> ", PACKAGE,
+	size = snprintf(buf, sizeof(buf), "%s:%d:%s:%s:%s():%d<%s> ", PACKAGE,
+			getpid(),
```

Comment:
Please separate this change into it's own PR, so we can flag it for the other provider maintainers to review.  We don't know what tools may be processing log output.  Thanks!

Reply:
Yes, it would be good to have this in separate PR. 

Also it would be helpful to control the output through some env variable. We may not need to log PIDs all the time. So an env variable like FI_LOG_FIELDS="pid,hostname,etc" can allow logging additional fields. This can be done as an enhancement at a later time as well.

Reply:
I didn't want to include the log-related changes - it was added just for debugging purposes.
I agree to drop these changes for now.  @a-ilango  I vote for you proposal, it looks reasonable.